src/test/java/seedu/address/model/client/LastModifiedInstantTest.java

```diff
@@ -0,0 +1,122 @@
+package seedu.address.model.client;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.time.LocalDate;
+
+import org.junit.jupiter.api.Test;
+
+class LastModifiedInstantTest {
+
+    private static final String VALID_INSTANT_1 = "2020-10-10T15:18:35.617617Z";
+    private static final String VALID_INSTANT_2 = "2020-10-11T15:18:35.617617Z";
+
```

Comment:
Can probably add one more test for the empty constructor then check if Instant.now() is < 1s away from the constructed LastModifiedInstant.

Reply:
Placed it together with the existing test.